.circleci/config.yml

```diff
@@ -407,6 +407,15 @@ jobs:
       - run: docker run --privileged linuxkit/binfmt:v0.6
       - run: cp dist/grafana-latest.linux-*.tar.gz packaging/docker
       - run: cd packaging/docker && ./build.sh --fast "${CIRCLE_SHA1}"
+      - run:
+          name: Archive Docker image
+          command: |
+            mkdir -p artifacts
+            docker save -o artifacts/docker-image-${CIRCLE_SHA1}.tar grafana/grafana:${CIRCLE_SHA1}
+            ls -al artifacts
```

Comment:
Is this left over from debugging?

Reply:
the `ls -al` is left over, have the docker image saved to artifacts made testing this really easy.  it can be removed though if it is a problem.

Reply:
removed in newest commit